src/main/webapp/readfile.html

```diff
@@ -15,57 +15,25 @@ <h2>Read Configuration</h2>
       <a href="deletefile.html">Delete Configuration</a>
     </div>
     <div>
-    <form id="config-search" action="/BillingConfig" method="get">
+    <form id="config-search" action="javascript:addToPage()">
       <p>Customer ID:</p>
       <input list="customer-ids">
       <datalist id="customer-ids">
+        <option value="sample">
       </datalist>
       <p>Account ID:</p>
       <input list="account-ids">
       <datalist id="account-ids">
         <option value="Account 1">
         <option value="Account 2">
         <option value="Account 3">
+        <option value="account-id">
```

Comment:
What happens if a user clicks on this option?

Reply:
No support for searching for specific accounts yet.

Reply:
#42